pkg/integration/tests/commit/commit_with_non_matching_branch_name.go

```diff
@@ -30,9 +30,6 @@ var CommitWithNonMatchingBranchName = NewIntegrationTest(NewIntegrationTestArgs{
 
 		t.ExpectPopup().CommitMessagePanel().
 			Title(Equals("Commit summary")).
-			/* EXPECTED:
 			InitialText(Equals(""))
-			ACTUAL: */
-			InitialText(Equals("branchnomatch"))
```

Comment:
I took the liberty of changing the `//` to `/*` because I like the resulting diff in the second commit a bit more. It's just my habit to write it like this.